include/seeds.proposals.hpp

```diff
@@ -111,7 +117,8 @@ CONTRACT proposals : public contract {
           uint64_t proposal_id;
           name account;
           uint64_t amount;
-          bool favour;
```

Comment:
Question: Will this impact existing stored votes tables? Will they become unreadable?

Is there a solution to migrate old votes tables over, or would it be better to simply use a new type name?

@IanMendozaJaimes